apps/frontend/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_ANALYTICS=GTM-PX36D3G
```

Comment:
Leave value empty here, or rename `.env` to `.env.example`, or move this env to `netlify.toml`.

Current approach will trigger events in development mode and we do not want it, having it in the `netlify.toml` file you will be able to target branches with different gtag id's.

Example: https://github.com/DistributedCollective/Sovryn-frontend/blob/development/netlify.toml#L21